src/main/java/wooteco/subway/line/dao/LineDao.java

```diff
@@ -48,13 +57,25 @@ public Line findById(Long id) {
                 "left outer join STATION DST on S.down_station_id = DST.id " +
                 "WHERE L.id = ?";
 
-        List<Map<String, Object>> result = jdbcTemplate.queryForList(sql, new Object[]{id});
-        return mapLine(result);
+        try {
+            List<Map<String, Object>> result = jdbcTemplate.queryForList(sql, new Object[]{id});
```

Comment:
```suggestion
            List<Map<String, Object>> result = jdbcTemplate.queryForList(sql, id);
```
마지막 매개변수가 가변인자라 이렇게도 가능합니다 :)

Reply:
그냥도 가능하군요~!